packages/sign_in_with_apple/lib/src/widgets/sign_in_with_apple_button.dart

```diff
@@ -102,12 +183,35 @@ class SignInWithAppleButton extends StatelessWidget {
   }
 }
 
+/// The style of the button according to the Apple Documentation.
+///
+/// https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/buttons/
 enum SignInWithAppleButtonStyle {
-  dark,
+  /// A black button with white text and white icon
+  ///
+  /// ![Black Button](https://raw.githubusercontent.com/aboutyou/dart_packages/master/packages/sign_in_with_apple/test/sign_in_with_apple_button/goldens/black_button.png)
```

Comment:
If these works, that'd be great :)

Reply:
I would hope so, I saw atleast images in the flutter docs and those worked